lms/static/scripts/ui-playground/components/ErrorComponents.js

```diff
@@ -100,221 +101,115 @@ function OAuth2RedirectErrorAppExample({ errorCode = '' }) {
   }
 }
 
+function LaunchErrorDialogExample({ errorState = '' }) {
```

Comment:
I am unable to close the modal, is this intended? (I had to reload the page).

Reply:
I added instructions about how to close.